crates/core/component/stake/src/validator/definition.rs

```diff
@@ -29,11 +29,27 @@ impl From<Definition> for pb::ValidatorDefinition {
 impl TryFrom<pb::ValidatorDefinition> for Definition {
     type Error = anyhow::Error;
     fn try_from(v: pb::ValidatorDefinition) -> Result<Self, Self::Error> {
+        let validator = v
+            .validator
+            .ok_or_else(|| anyhow::anyhow!("missing validator field in proto"))?;
+        // Validation:
+        // - Website has a max length of 70 chars
+        if validator.website.len() > 70 {
+            anyhow::bail!("validator website field must be less than 70 characters");
+        }
+
+        // - Name has a max length of 140 chars
+        if validator.name.len() > 140 {
+            anyhow::bail!("validator name must be less than 140 characters");
+        }
+
+        // - Description has a max length of 280 chars
+        if validator.description.len() > 280 {
+            anyhow::bail!("validator description must be less than 280 characters");
+        }
+
```

Comment:
I don't think we need/want this because we construct a `Validator` down below which is where the validation occurs

Reply:
Since the `Definition` can get included on-chain (as part of `Action::ValidatorDefinition`) it seems important to apply a length limit to it.